### PR TITLE
Kv tutorial updates

### PR DESCRIPTION
**Pull request by ipopescu**
Added some improvements and fixed the formatting and broken links.

### CODE REVIEW

File: dapp-dev-guide/tutorials/kv-storage-tutorial.rst [hunk 1]
```diff
@@ -245,28 +243,38 @@ When working with the testnet, create an account on `Testnet CSPR Live <https://
...
-   ```bash
-   casper-client put-deploy --session-name kvstorage_contract --session-entry-point store-string --session-arg=name:"string=`test`" --payment-amount 100000000000 --chain-name <CHAIN-NAME> --node-address http://<HOST>:<PORT> --secret-key /home/keys/secretkey.pem
+    casper-client put-deploy 
+        --session-name kvstorage_contract 
+        --session-entry-point store-string 
+        --session-arg=name:"string=`test`" 
+        --payment-amount 100000000000 
+        --chain-name <CHAIN-NAME> 
+        --node-address http://<HOST>:<PORT> 
+        --secret-key /home/keys/secretkey.pem
```
Comment (ramujin) [inline dapp-dev-guide/tutorials/kv-storage-tutorial.rst:275]: Also a note about the path here.

**Review by ramujin — COMMENTED** [review]
Overall everything looks great, but there is a rendering issue with the code block and I made a few notes about mentioning when paths are examples (e.g., for the session path or the key path).

File: dapp-dev-guide/tutorials/kv-storage-tutorial.rst [hunk 2]
```diff
@@ -245,28 +243,38 @@ When working with the testnet, create an account on `Testnet CSPR Live <https://
...
+After compiling the contract, you need to deploy the compiled WASM to the network. This action installs the contract in the blockchain.
+Once the contract is deployed, you can use the Casper client to retrieve the contract session hash and the block hash where the contract is deployed.
 
-```casper-client put-deploy --chain-name :raw-html-m2r:`<CHAIN-NAME>` --node-address http://\ :raw-html-m2r:`<HOST>`\ :\ :raw-html-m2r:`<PORT>` --secret-key /home/keys/secretkey.pem --session-path  $HOME/kv-storage-contract/target/wasm32-unknown-unknown/release/contract.wasm  --payment-amount 1000000000000
+.. code-block:: bash
 
-.. code-block::
+    casper-client put-deploy 
+        --chain-name :raw-html-m2r:`<CHAIN-NAME>` 
+        --node-address http://\ :raw-html-m2r:`<HOST>`\ :\ :raw-html-m2r:`<PORT>`
```
[the remark this answers is not in the document]
Comment (ipopescu) [reply dapp-dev-guide/tutorials/kv-storage-tutorial.rst]: Thanks, Ramsin! I missed this and fixed it with commit 1ab09f7.

File: dapp-dev-guide/tutorials/kv-storage-tutorial.rst [hunk 3]
```diff
@@ -245,28 +243,38 @@ When working with the testnet, create an account on `Testnet CSPR Live <https://
...
 
-```casper-client put-deploy --chain-name :raw-html-m2r:`<CHAIN-NAME>` --node-address http://\ :raw-html-m2r:`<HOST>`\ :\ :raw-html-m2r:`<PORT>` --secret-key /home/keys/secretkey.pem --session-path  $HOME/kv-storage-contract/target/wasm32-unknown-unknown/release/contract.wasm  --payment-amount 1000000000000
+.. code-block:: bash
 
-.. code-block::
+    casper-client put-deploy 
+        --chain-name :raw-html-m2r:`<CHAIN-NAME>` 
+        --node-address http://\ :raw-html-m2r:`<HOST>`\ :\ :raw-html-m2r:`<PORT>` 
+        --secret-key /home/keys/secretkey.pem 
+        --session-path  $HOME/kv-storage-contract/target/wasm32-unknown-unknown/release/contract.wasm
```
[the remark this answers is not in the document]
Comment (ipopescu) [reply dapp-dev-guide/tutorials/kv-storage-tutorial.rst:256]: Very good catch. Addressed with commit 6f9206f.

File: dapp-dev-guide/tutorials/kv-storage-tutorial.rst (hunk 1 above)
Comment (ipopescu) [reply dapp-dev-guide/tutorials/kv-storage-tutorial.rst]: Addressed with commit 6f9206f.

**Review by ramujin — APPROVED** [review]
Looks great to me now! Great work!

**Comment by ipopescu** [comment]
> Looks great to me now! Great work!

Thanks @ramujin!

**Pull request merged**